src/main/java/com/ericsson/ei/subscription/SubscriptionValidator.java

```diff
@@ -76,10 +76,15 @@ public static void validateSubscription(Subscription subscription) throws Subscr
      * @param subscriptionName
      */
     private static void validateSubscriptionName(String subscriptionName) throws SubscriptionValidationException {
-        String regex = "^[A-Za-z0-9_]+$";
+        // When this regExp need to be changed then remember to change the one in the
```

Comment:
Maybe we should discuss in the team how we would like to handle regexp's. Maybe it should be added in the Eiffel Commons repository so the same regexp's may be loaded from there, Maybe even have them in a property file like emailValidationRegex and then any Eiffel products may use the same regexp.

Reply:
Eiffel-Common is good place the validation code.
Regexp source should be mapped/defined in som Json Map file.
It would be good if Frontend/bridge has a rest entrypoint so JavaScript code in web-browser can use same Regexp for validations in Javascript code.

Reply:
Yes, I agree we need a discussion.... As per bug requirement, it is now same in both backend and frontend. So I intend to create a story after the discussion.